CHANGELOG.md

```diff
@@ -0,0 +1,24 @@
+# Changelog
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
+and this project adheres to SFCC versioning practice where the Year is the
+first number and the Month is the second number. The third number will be
+bumped for multiple releases during one month.
+
+### [Unreleased]
+
+### [20.11.0] - 2020-11-25
+##### Added
+- Enable Prophet debugger to detect cartridge.
+- Enable watch and upload cartridge with sgmf cli tool.
+- Fix image size and email that can crash if empty.
+- Add trackViewedItem for profile tracking.
+
+[Unreleased]: https://github.com/SalesforceCommerceCloud/link_klaviyo/compare/HEAD...release-20.11.0
```

Comment:
I think HEAD and the release tag are switched but maybe not. I cannot see this repo: https://github.com/SalesforceCommerceCloud/link_klaviyo

Which brings me to my other thought, is this changelog for this repo or for the repo linked?

Reply:
it should be for this repo

Reply:
We were under the assumption it should be for both - should I make a new changelog for this private one?

Reply:
I'm not familiar with the relationship between the various repos but in general I think each one should have its own separate changelog

Reply:
What would the structure of that look like for this private repo? I ask because we haven't tagged any releases in this one, only on the official linked one.